### PR TITLE
Re implement sand options history and update

### PR DESCRIPTION
**Pull request by GianfrancoIngratta**
The last merge eliminated the history of previous versions of the SAND geometry. Although the default version is unchanged from the last merge, here we want to restore the other options as well so that we can keep track of the evolution of the SAND geometry.

### CODE REVIEW

**Review by ast0815 — CHANGES_REQUESTED** [review]
Hi, this looks fine overall. But could you please remove the changes that introduce extra white spaces in empty lines, like line 148 in SAND_DRIFT.py?

**Comment by GianfrancoIngratta** [comment]
not sure if you are referring to the right line number, but ok I think I got rid of some empty lines both in SAND_DRIFT.py and STT.py.
Moreover, I removed a config file "SAND_INNERVOLOPTDRIFT.cfg" that was previously renamed in  "SAND_INNERVOLOPTDRIFT1.cfg" and became useless.

File: duneggd/SubDetector/SAND_DRIFT.py [hunk 1]
```diff
@@ -144,7 +145,7 @@ def FillTracker(self, geom, volume):
     def constructExtraMod(self, geom, volume):
          
         running_x = self.kloeVesselRadius - self.clearenceTrackerECAL
-
+
```
Comment (ast0815) [inline duneggd/SubDetector/SAND_DRIFT.py:148]: I mean lines like this. You removed an empty line, and added 4 spaces.

**Comment by ast0815** [comment]
You don't have to remove empty lines. Just don't add any whites paces in lines that were empty before.

**Comment by ast0815** [comment]
In general, it is good practice to avoid unrelated changes in white spaces like adding or removing empty lines or trailing white spaces (white spaces at the end of lines).

But actual edits of lines (where you change the code) should never introduce trailing white spaces, and remove them if they were there before. You can configure your `git diff` to highlight trailing white spaces for you.

**Comment by GianfrancoIngratta** [comment]
I see your point now. 
In my new commit, I replaced the empty lines with some spaces introduced in the first commit with just empty lines. 
Hope this was the correct procedure to follow.

File: duneggd/SubDetector/SAND_DRIFT.py [hunk 2]
```diff
@@ -380,7 +375,7 @@ def placeSubVolume(self, geom, volume, subvolume, pos_x=Q("0mm"), pos_y=Q("0mm")
         position = geom.structure.Position(name + "_pos", pos_x, pos_y, pos_z)
         rotation = geom.structure.Rotation(name + "_rot", rot_x, rot_y, rot_z)
         place    = geom.structure.Placement(name + "_place", volume = subvolume.name, pos = position.name, rot = rotation.name)
-
+
```
Comment (ast0815) [inline duneggd/SubDetector/SAND_DRIFT.py:378]: This line still introduces trailing white spaces.

**Review by ast0815 — APPROVED** [review]
Now you have introduced lots of white space changes that have nothing to do with your actual code changes. This is not good practice, but good enough, since it only changes things for the better.

**Pull request merged**